app/src/main/java/de/xikolo/controllers/section/VideoPreviewFragment.kt

```diff
@@ -103,6 +103,11 @@ class VideoPreviewFragment : ViewModelFragment<VideoPreviewViewModel>() {
             }
     }
 
+    override fun onResume() {
+        super.onResume()
+        viewModel.onRefresh()
```

Comment:
I checked again and this piece of code is not relevant anymore. It solved the problem of re-loading the DownloadViewHelpers when the fragment is resumed but is now not used anymore because they hold a reference to a listener object. Thanks for the catch!

Reply:
Even better. :)